src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -230,13 +230,14 @@ public static string SelectProductNameForDirectory(Platform.XDG_Type dirpath)
         {
             //TODO: XDG_DATA_DIRS implementation as per GitHub issue #1060
 
+            string defaultHomePath = System.Environment.GetEnvironmentVariable("HOME") ?? Directory.GetCurrentDirectory();
```

Comment:
About PermissionsStartOnly, https://www.freedesktop.org/software/systemd/man/systemd.service.html shows:

>PermissionsStartOnly=
Takes a boolean argument. If true, the permission-related execution options, as configured with User= and similar options (see systemd.exec(5) for more information), are only applied to the process started with ExecStart=, and not to the various other ExecStartPre=, ExecStartPost=, ExecReload=, ExecStop=, and ExecStopPost= commands. If false, the setting is applied to all configured commands the same way. Defaults to false.